src/tag/src/Tag.tsx

```diff
@@ -195,7 +205,12 @@ export default defineComponent({
         {!this.checkable && this.closable ? (
           <NBaseClose
             clsPrefix={mergedClsPrefix}
-            class={`${mergedClsPrefix}-tag__close`}
+            class={[
+              `${mergedClsPrefix}-tag__close`,
+              {
+                [`${mergedClsPrefix}-tag__close--rtl`]: rtlEnabled
```

Comment:
use class from outermost dom to apply style

Reply:
@07akioni done